test/small/APIAuthorStateTest.ts

```diff
@@ -95,6 +138,117 @@ describe("Test APIAuthor#getState", function() {
         })
     })
 })
+describe("Test APIAuthor#queryStates", function() {
```

Comment:
Please test with test class, so that can be shorter and improved reusability. Like this test case https://github.com/KiiPlatform/thing-if-JSSDK/blob/develop/test/small/OnboardingOpsTest.ts#L179